cmd/controller/main.go

```diff
@@ -230,9 +231,9 @@ func main() {
 	}
 }
 
-func run(ctrl *controller.Impl, threadiness int) doneFunc {
+func run(ctx context.Context, ctrl *controller.Impl, threadiness int) doneFunc {
```

Comment:
are these `run` changes correct? @tomkennedy513 @matthewmcnew

Reply:
I think perhaps this could simplify it? https://github.com/pivotal/kpack/pull/1011